automatic/gitextensions/gitextensions.nuspec

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>gitextensions</id>
+    <version>2.51.02</version>
+    <title>Git Extensions</title>
+    <authors>Henk Westhuis</authors>
+    <owners>chocolatey</owners>
+    <licenseUrl>https://github.com/gitextensions/gitextensions/blob/master/LICENSE.md</licenseUrl>
+    <projectUrl>http://gitextensions.github.io/</projectUrl>
+    <iconUrl>https://cdn.rawgit.com/ferventcoder/chocolatey-packages/02c21bebe5abb495a56747cbb9b4b5415c933fc0/icons/gitextensions.svg</iconUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description><![CDATA[## Features
+
+* Windows Explorer integration for Git
+* Feature rich user interface for Git
+]]></description>
+    <summary>Git Extensions is a graphical user interface for Git that allows you to control Git without using the commandline.</summary>
+    <releaseNotes>https://github.com/gitextensions/gitextensions/blob/master/GitUI/Resources/ChangeLog.md</releaseNotes>
+    <tags>foss gitextensions git gui admin</tags>
+    <projectSourceUrl>https://github.com/gitextensions/gitextensions</projectSourceUrl>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/automatic/gitextensions</packageSourceUrl>
+    <docsUrl>https://git-extensions-documentation.readthedocs.org/en/latest/</docsUrl>
+    <mailingListUrl>http://groups.google.com/group/gitextensions</mailingListUrl>
+    <bugTrackerUrl>http://github.com/gitextensions/gitextensions/issues</bugTrackerUrl>
+    <dependencies>
+      <dependency id="git" />
+      <dependency id="chocolatey-core.extension" />
+    </dependencies>
+  </metadata>
+</package>
```

Comment:
please add the files element, and add the `legal` and `tools` directory to it.
By leaving it empty it will package every file in this directory and sub-directories relative to the nuspec file, meaning it will pack a few files it shouldn't.